### PR TITLE
Wazo 2656 meeting limit api

### PR DESCRIPTION
**Pull request by pc-m**
Depends-On: https://github.com/wazo-platform/wazo-calld-client/pull/45

### CODE REVIEW

**Comment by bloom1** [comment]
recheck

**Comment by pc-m** [comment]
recheck

File: wazo_calld/plugins/meetings/api.yml [hunk 1]
```diff
@@ -1,4 +1,21 @@
 paths:
+  /guests/me/meetings/{meeting_uuid}/status:
```
Comment (bloom1) [inline wazo_calld/plugins/meetings/api.yml:2]: The CHANGELOG entry is missing for this new route

File: integration_tests/assets/etc/wazo-calld/conf.d/20-real-asterisk.yml [hunk 2]
```diff
@@ -2,3 +2,4 @@ ari:
   connection:
     username: xivo
     password: xivo
+max_meeting_participants: 4
```
Comment (bloom1) [inline integration_tests/assets/etc/wazo-calld/conf.d/20-real-asterisk.yml:5]: Is that enforced somewhere else or am I missing something? 
Should that be in the wazo-confd API and defined for each meeting? I'm not sure I understand

**Review by bloom1 — CHANGES_REQUESTED** [review]

File: integration_tests/assets/etc/wazo-calld/conf.d/20-real-asterisk.yml (hunk 2 above)
Comment (pc-m) [reply integration_tests/assets/etc/wazo-calld/conf.d/20-real-asterisk.yml:5]: In `config.py` it is set to 25 which is the current "advertised" limit. I made it a configuration parameter to be able to test the limit without using 26 calls and to be able to change that limit more easily if/when we decide that 25 is not the limit anymore.

**Review by bloom1 — APPROVED** [review]

**Comment by pc-m** [comment]
regate

**Pull request merged**